storage/innobase/handler/ha_innodb.cc

```diff
@@ -12167,6 +12173,16 @@ bool create_table_info_t::create_option_encryption_is_valid() const {
     return (false);
   }
 
+  if (!table_is_keyring &&
+      Encryption::is_master_key_encryption(m_create_info->encrypt_type.str) &&
+      Encryption::is_online_encryption_on()) {
+    my_printf_error(ER_ILLEGAL_HA_CREATE_OPTION,
+                    "InnoDB: ENCRYPTED='Y' not supported for table because "
+                    "online encryption to KEYRING is turned ON.",
```

Comment:
I feel mentioning the variable name here is better and user need not search for the variable to fix. 

My suggestion is "because innodb_encrypt_tables=get_string(srv_encrypt_tables)"

Reply:
Ok to have both too (above text and this)